hibernate-core/src/main/java/org/hibernate/bytecode/internal/bytebuddy/ByteBuddyState.java

```diff
@@ -57,7 +57,7 @@ public final class ByteBuddyState {
 
 	private static final MethodHandles.Lookup LOOKUP = MethodHandles.lookup();
 
-	private static final boolean DEBUG = false;
+	private static final boolean DEBUG = true;
```

Comment:
```suggestion
	private static final boolean DEBUG = false;
```

Don't forget to revert that ;)

Reply:
Oh 😅 thank you for catching that 🙏

Reply:
there's no checkstyle in ORM anymore, rigth? but maybe you can do something similar with spotless. In Search, we have e.g.: 

https://github.com/hibernate/hibernate-search/blob/79dc470d7105b5abed29ad9e6be3bc1ef6a8db69/build/config/src/main/resources/checkstyle.xml#L44-L49

to prevent committing these "debug" changes 😄